### DISCUSSION TITLE
How to not include other subdomains in notification?

### DISCUSSION THREAD

**Discussion by 0xspade** [projectdiscovery/interactsh#158] [Q&A] [answered]
Hi Team,

I have been bugging this noisy notification with the queries (www.interactdomain.sh, ns1.interactdomain.sh, ns2.interactdomain.sh, interactdomain.sh)

Tried prepping like

`interactsh-client -server https://interactdomain.sh -token XXX | grep -vE "\[interactdomain.sh\]|\[www.interactdomain.sh\]|\[ns1.interactdomain.sh\]|\[ns2.interactdomain.sh\]" | notify`

but there are no outputs. 

Are there any alternatives on this one?

**Comment by ehsandeep** [comment, marked as answer]
```
grep -vE "\[www.interactdomain.sh\]|\[ns1.interactdomain.sh\]|\[ns2.interactdomain.sh\]" | notify
```

looks good to me, but not sure it's a good idea to pipe the global interaction to `notify` as any known user can flood the notification anyway.

**Comment by 0xspade** [comment]
no notification on this one on my side.
